keyboards/projectkb/signature65/rules.mk

```
@@ -0,0 +1,20 @@
+# MCU name
+MCU = STM32F072
+
+# Bootloader selection
+BOOTLOADER = stm32-dfu
+
+# Build Options
+#   comment out to disable the options.
+#
+#BOOTMAGIC_ENABLE = full	# Virtual DIP switch configuration
```

There is no "full" bootmagic anymore, only the "lite" option, which can be turned on with just "yes".
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```